def handleError(self, test, err):
    '\n        Adds Exception throwing test cases and information to log.\n        '
    self.tcRunLogger.fatal(('%s: %s: %s' % (EXCEPTION, self.testName, GetDetailExceptionInfo(err))))
    self.testResult = EXCEPTION
